data/jobs.txt

```diff
@@ -1379,6 +1379,80 @@ mission "Bounty Hunting (Small)"
 		payment 150000
 		dialog "The government of <planet> gratefully pays you <payment> for eliminating the <npc>."
 
+mission "Bounty Hunting (Small, boarding, entering)"
+	name "Board <npc>"
+	description "A wanted criminal is fleeing in a stolen hot rod ship, the <npc>, which is about to enter this system. Disable and board the vessel and bring the criminal to <destination>. Reward for capture: <payment>."
+	passengers 1
+	repeat
+	job
+	to offer
+		"combat rating" > 80
+		random < 15
+	on offer
+		require "Brig"
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes rim south north "dirt belt" core frontier
+	destination
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes rim south north "dirt belt" core frontier
+		distance 2 6
+	npc board
+		government "Bounty"
+		personality timid fleeing uninterested entering target
```

Comment:
Timid doesn't do much here as uninterested single-ship NPCs have no parent to stay close to

Reply:
Are you thinking of "coward?" AFAIK "timid" isn't affected by fleet status.

Reply:
A code summary for personalities used:
`timid`: 
 - If further away than 500px from your parent ship, get closer to them, even if you're fighting (and even if you're `heroic`
 -  If the combat target is disabled and further away than 1000px, find a new target

`fleeing`
 - when health is low, consider abandoning a fight
 - when landing, don't take off again until the player takes off again (altered to never take off again in #2990 )

`uninterested`
 - Do not make movement decisions based on the player, ever

`entering` and `target` have no behavioral impacts
`coward`:
 - when health is low, abandon your parent

I've asked for clarification in the wiki on what these personality tokens do and what they are supposed to do in #2658 (and also shared what they currently do).

Reply:
Most of the existing uses of `timid` are applied to single ships right now, implemented by @endless-sky himself. I'm going to leave `timid` for now to follow the trend, and in case in the future the behavior is changed to better reflect what the word and wiki page suggest.

Reply:
If those single ship NPCs are not `uninterested`, then there is an impact because they have the player as a parent ship. They may also have `timid` for the combat targeting aspect, where they forget about disabled ships once disabled ships are far enough away.

What is your goal for these ships' behavior? How should they attack, or should they always run and never attack?

If always run and never attack, give them `pacifist uninterested` (and possibly `fleeing` if you don't want them to take off again when they land to refuel).

Reply:
@Pointedstick this comment chain is my only remaining feedback. Consider it optional